### DISCUSSION TITLE
Shutdown function

### DISCUSSION THREAD

**Discussion by GreyEarl** [Ludy87/xplora_watch#109] [Ideas]
Hi there,

I see that your Xplora library supports shutdown(wuid: str) function. Do you have plans to integrate in the custom component?

Use case sample: 

When after X time AND is connected to home WiFi run shutdown command to save battery until next day of usage.

Thanks!

**Comment by Ludy87** [comment]
Providing this as a service shouldn't be a problem.

**Comment by Ludy87** [comment]
FYI: https://github.com/Ludy87/xplora_watch/releases/tag/v2.5.0

**Reply by GreyEarl** [reply]
Wow! Didn't expect you that fast! Awesome, thank you very much, appreciate it!!
